memeify/memeify.py

```diff
@@ -56,11 +69,216 @@ def __bify(self, bify_str, cmd) -> str:
         return "".join(b)
 
     def __bify_f(self, bif) -> str:
-        vowel = re.compile("^[aeiouAEIOU]")
         if vowel.match(bif) and len(bif) > 1:
             # adds b in front of the word
             bif = "b" + bif
         elif len(bif) > 1:
             # replaces first letter with b
             bif = "b" + bif[1:]
         return bif.replace("b", "🅱️")
+
+    # ------------ frenchify ------------
+    @commands.command()
+    async def frenchify(self, ctx, *, content: str = None):
+        """Writes a message with a french accent"""
+        msg_c = await self.__get_content(ctx, content)
+        if len(msg_c) != 2:
+            await ctx.send("No message")
+        else:
+            await ctx.send(self.__french_pre_f(*msg_c))
+
+    def __french_pre_f(self, french, cmd):
+        emoji_list = []
+        if cmd:
+            french_cmd_fix = french.split(" ")
+            french_str = " ".join(french_cmd_fix[1:])
+
+        emoji_match = re.finditer(custom_emoji, french_str)

```

Comment:
Big error. `french_str` can be referenced before assignment if cmd is not none. You need an else for this statement to sent french_str (or move this under the if statement.

Reply:
Happens when you run frenchify with no text.